### PR TITLE
Malay Translations

### PR DESCRIPTION
**Pull request by plokmi**
I would like to contribute for Malay translations specifically.
Added links to be merged to the main repo.

you can contact me @ #plokmi5120 on discord or plokmi@live.com.my.

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "code": "en",
-  "language": "English",
-  "maintainers": ["schacon"],
+  "code": "ms",
```
Comment (jnavila) [inline status.json]: You don't need to update the status.json file. It is no longer used.

Comment (plokmi) [reply status.json]: Alright, noted. Will revert the change.

**Comment by jnavila** [comment]
OK. I checked your translation repo. You have to translate at least the first chapter. Otherwise, that does not mean anything to list your repo.

**Comment by plokmi** [comment]
Will do, will inform here asap once I've done at least the first chapter. Just would like to set things up.

**Comment by plokmi** [comment]
Close for the moment.

**Comment by dbucci24** [comment]
@jnavila hey, i want to translate this project into spanish.. how can i know which ones need translation and which are already translated? i hope your answer, thanks :)

**Comment by jnavila** [comment]
@dbucci24 You can refer to the [CONTRIBUTING](https://github.com/progit/progit2/blob/master/CONTRIBUTING.md) document where you'll see that there's already a [project](https://github.com/progit/progit2-es) :smiley:

**Comment by dbucci24** [comment]
@jnavila ok, thank you.. is this project in Crowdin?

**Comment by jnavila** [comment]
I don't know. You'd better ask the maintainers ; it seems to me that the book is already totally translated (see https://git-scm.com/book/es) but it may need some backports of recent changes.

**Pull request closed without merging**